share/templates/01-General/01-Treble_Clef.mscx

```diff
@@ -3,12 +3,8 @@
   <Score>
     <LayerTag id="0" tag="default"></LayerTag>
     <currentLayer>0</currentLayer>
-    <Synthesizer>
-      </Synthesizer>
     <Division>480</Division>
     <Style>
-      <enableVerticalSpread>1</enableVerticalSpread>
```

Comment:
Same here (and on also same for the other templates).

Reply:
same here